support/android/apk/build.gradle

```diff
@@ -2,8 +2,8 @@ import java.text.SimpleDateFormat
 
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id 'com.android.application' version '8.0.1' apply false
```

Comment:
I think upgrading the AGP will also moving to a newer build tools version. On my NixOS system, this manifests as an error:
```
License for package Android SDK Build-Tools 34 accepted.
Preparing "Install Android SDK Build-Tools 34 v.34.0.0".
> Failed to install the following SDK components:
      build-tools;34.0.0 Android SDK Build-Tools 34
  The SDK directory is not writable (/nix/store/b5p83nwz8f22w5mgqizz8s7yrrif37jj-androidsdk/libexec/android-sdk)
```

We should also update the build-tools version in [README.md](https://github.com/servo/servo/blob/f8933a57353aeca14a6cbc60b3cb0cf98cab6c5d/README.md?plain=1#L64) and [shell.nix](https://github.com/servo/servo/blob/f8933a57353aeca14a6cbc60b3cb0cf98cab6c5d/etc/shell.nix#L38)

Reply:
Addressed with [5753e81](https://github.com/servo/servo/pull/33681/commits/5753e811191e28b32c1da25f04704ca089a837fb)

Reply:
Does it work now for NixOS?

Reply:
No, nixpkgs doesn't have build tools 35.0.0 yet, there are only RC versions of v35. I tried with "34.0.0" and that does work. [This table from AGP 8.7 release notes](https://developer.android.com/build/releases/gradle-plugin#compatibility) indicates that 34.0.0 is the default (though the text above the table refers to v8.6 of AGP, which I think is just a typo in the release notes), so maybe we can go with "34.0.0"?

Reply:
I swear I saw it in Android Studio as 35.0.0, I'll correct things tonight.

Reply:
Corrected to 34.0.0, may you try again?

Reply:
34.0.0 works for me! Thank you!